Kitsune/ClubPenguin/Login.php

```diff
@@ -27,14 +27,15 @@ protected function handleLogin($socket) {
 		$password = Packet::$Data['body']['login']['pword'];
 		
 		if($penguin->database->usernameExists($username) === false) {
+			password_verify('KitsuneTimingAttack', '$2y$12$5hf7vOtBjsNJf6oZSsnyXOlcLTRblTwSg550SR0ohxVUkItmXhMM6'); //Mitigate timing attacks
```

Comment:
I'm a bit confused. If the username doesn't exist, password_verify is never called, and therefore it will take less time if the username doesn't exist than if the password for the account is incorrect.

My understanding is password_verify makes it so you cannot distinguish what the correct password or its hash is from a timing attack, but the only way it could mitigate a timing attack against non-existent accounts is if password_verify literally takes no clock time to run.